www/sidebars.js

```diff
@@ -94,7 +99,7 @@ module.exports = {
       collapsed: false,
       link: {
         type: 'generated-index',
-        title: '@trpc/react-query guides',
+        title: 'tRPC React documentation',
```

Comment:
Should be React Query I think?
```suggestion
        title: 'tRPC React Query documentation',
```